src/js/menu/menu.jsx

```diff
@@ -69,24 +79,25 @@ var NestedMenuItem = React.createClass({
   },
 
   _onParentItemClick: function() {
-    this.setState({ open: !this.state.open });
+    if (!this.props.disabled) this.setState({ open: !this.state.open });
   },
 
   _onMenuItemClick: function(e, index, menuItem) {
-    this.setState({ open: false });
     if (this.props.onItemClick) this.props.onItemClick(e, index, menuItem);
+    this.setState({ open: false });
```

Comment:
That makes sense. If you could please remove the console log on line 96 then it should be good to go.